target/linux/ar71xx/files/drivers/net/ethernet/atheros/ag71xx/ag71xx_ar7240.c

```diff
@@ -943,6 +943,13 @@ ar7240_hw_apply(struct switch_dev *dev)
 			portmask[i] = 1 << AR7240_PORT_CPU;
 			portmask[AR7240_PORT_CPU] |= (1 << i);
 		}
+		/* Overwrite isolate and allow port-to-port forward */
```

Comment:
This is superfluous anyway. You shouldn't make the code write a value and immediately override it. Just rewrite the for loop above to accomplish what you want.

Reply:
that is ok but.

so I just add some code to overwrite. it is more elegant and easy to read and easy to restore original, I think.